cmd/vic-machine/common/target.go

```diff
@@ -56,6 +58,16 @@ func (t *Target) TargetFlags() []cli.Flag {
 			Value: flags.NewOptionalString(&t.Password),
 			Usage: "ESX or vCenter password",
 		},
+		cli.BoolTFlag{ // TODO: s/BoolTFlag/BoolFlag/ - should be false by default
+			Name:        "insecure",
```

Comment:
If we specify a thumbprint at all, doesn't that imply --insecure=false? This could be added to the processCertificates method.